superset-frontend/src/explore/components/controls/DndColumnSelectControl/utils/optionSelector.ts

```diff
@@ -33,10 +33,10 @@ export class OptionSelector {
     let groupByValues: string[];
     if (Array.isArray(values)) {
       groupByValues = values;
-      this.isScalar = false;
+      this.isArray = false;
```

Comment:
Shouldn't this be `this.isArray = true`?

Reply:
thanks for reminding me.